examples/color/color_cycle_default.py

```diff
@@ -1,4 +1,8 @@
 """
+======================================
+Colors in the default property cycle
+======================================
```

Comment:
It actually can be longer, but not shorter than the text, so this will compile fine.